src/main/java/spring/service/UserService.java

```diff
@@ -1,12 +1,13 @@
 package spring.service;
 
 import java.util.List;
+import java.util.Optional;
 import spring.model.User;
 
 public interface UserService {
     void add(User user);
 
     List<User> listUsers();
 
-    User get(Long id);
+    Optional<User> get(Long id);
```

Comment:
```suggestion
    User get(Long id);
```
We shouldn't return Optional on the service layer, only from dao